G_channels.md

```diff
@@ -249,6 +249,24 @@ socket = assign(socket, :user, msg["user"])
 
 Sockets store assigned values as a map in `socket.assigns`.
 
+#### Failure Semantics
```

Comment:
Instead of "Failure Semantics", which sounds a little grim, what would you think about, "Fault Tolerance" or "How Channels Handle Errors", or something else along those lines?